android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModule.java

```diff
@@ -692,15 +731,42 @@ private void cameraPickerResult(Activity activity, final int requestCode, final
             }
         }
     }
-
+    
+    
+    private void videoPickerResult(Activity activity, final int requestCode, final int resultCode, final Intent data) {
+        if (resultCode == Activity.RESULT_CANCELED) {
+            resultCollector.notifyProblem(E_PICKER_CANCELLED_KEY, E_PICKER_CANCELLED_MSG);
+        } else if (resultCode == Activity.RESULT_OK) {
+            Uri uri = mCameraCaptureURI;
+            
+            if (uri == null) {
+                resultCollector.notifyProblem(E_NO_IMAGE_DATA_FOUND, "Cannot resolve image url");
+                return;
+            }
+            
+            if (cropping) {
```

Comment:
UCrop support video cropping?
Are you sure JPEG compression can be set for video?